lib/prometheus-parser/Cargo.toml

```diff
@@ -19,3 +19,8 @@ vector-common = { path = "../vector-common", features = ["btreemap"] }
 
 [build-dependencies]
 prost-build = "0.12"
+
+[dependencies.tracing]
```

Comment:
Is this also used in other modules?

Reply:
yes, this tracing is used in multiple source, such as : https://github.com/databricks/vector/blob/60673c714dcc31e0f44fec00fcfa1a09fe437b8c/lib/codecs/Cargo.toml#L32